[2024-10-09 10:28:46,380] {taskinstance.py:1149} INFO - Dependencies not met for <TaskInstance: get_crypto_dag.AVAX scheduled__2024-09-16T00:00:00+00:00 [scheduled]>, dependency 'Task Instance State' FAILED: Task is in the 'scheduled' state.
[2024-10-09 10:28:46,482] {local_task_job.py:101} INFO - Task is not able to be run
[2024-10-09 10:29:37,094] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_crypto_dag.AVAX scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-10-09 10:29:37,805] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: get_crypto_dag.AVAX scheduled__2024-09-16T00:00:00+00:00 [queued]>
[2024-10-09 10:29:37,806] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-10-09 10:29:37,806] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-10-09 10:29:37,806] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-10-09 10:29:37,960] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): AVAX> on 2024-09-16 00:00:00+00:00
[2024-10-09 10:29:37,987] {standard_task_runner.py:52} INFO - Started process 52071 to run task
[2024-10-09 10:29:38,001] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'get_crypto_dag', 'AVAX', 'scheduled__2024-09-16T00:00:00+00:00', '--job-id', '353', '--raw', '--subdir', 'DAGS_FOLDER/get_crypto.py', '--cfg-path', '/tmp/tmp7_vqnvpy', '--error-file', '/tmp/tmpuz7gysj0']
[2024-10-09 10:29:38,007] {standard_task_runner.py:80} INFO - Job 353: Subtask AVAX
[2024-10-09 10:29:38,404] {task_command.py:370} INFO - Running <TaskInstance: get_crypto_dag.AVAX scheduled__2024-09-16T00:00:00+00:00 [running]> on host ken
[2024-10-09 10:29:38,559] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=get_crypto_dag
AIRFLOW_CTX_TASK_ID=AVAX
AIRFLOW_CTX_EXECUTION_DATE=2024-09-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-09-16T00:00:00+00:00
[2024-10-09 10:29:41,994] {history.py:122} ERROR - $AVAX: possibly delisted; no timezone found
[2024-10-09 10:29:52,011] {python.py:173} INFO - Done. Returned value was: None
[2024-10-09 10:29:52,057] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=get_crypto_dag, task_id=AVAX, execution_date=20240916T000000, start_date=20241009T132937, end_date=20241009T132952
[2024-10-09 10:29:52,136] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-10-09 10:29:52,444] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
